common/test/data/manual-testing-complete/problem/c2ee6e8917fe4c97ac99d7b616ff0b89.xml

```diff
@@ -73,7 +73,7 @@ while abs(balance) &gt; .02:
 
 # When the while loop terminates, we know we have 
 #  our answer!
-print "Lowest Payment:", round(payment, 2)
+print ( "Lowest Payment:", str( round(payment, 2) ) )
```

Comment:
I've ran the following code in a Python 3.8 interpreter and it works without the `str` conversion:

```sh
$ python3.8
>>> payment = 20.1123
>>> print("Lowest Payment:", round(payment, 2))
Lowest Payment: 20.11
```

So the code just needs the parenthesis:

```suggestion
print("Lowest Payment:", round(payment, 2))
```

Reply:
Thanks for the review @OmarIthawi 

I will push those changes and update the other file you linked as well.

Sorry for the delay, cheers!